pkg/platform/abstract/platform.go

```diff
@@ -344,3 +347,101 @@ func (ap *Platform) functionBuildRequired(createFunctionOptions *platform.Create
 	return false, errors.New("Function must have either spec.build.path," +
 		"spec.build.functionSourceCode, spec.build.image or spec.image set in order to create")
 }
+
+func (ap *Platform) GetProcessorLogsAndBriefError(scanner *bufio.Scanner) (string, string) {
+	var formattedProcessorLogs, briefErrorMessage string
+
+	for scanner.Scan() {
+		currentLogLine, logLevelAboveInfo, err := ap.prettifyProcessorLogLine(scanner.Bytes())
+		if err != nil {
+
+			// when it is unstructured just add the log as a text
+			formattedProcessorLogs += scanner.Text() + "\n"
+			briefErrorMessage += scanner.Text() + "\n"
+			continue
+		}
+
+		if logLevelAboveInfo {
+			briefErrorMessage += currentLogLine + "\n"
+		}
+
+		// when it is a processor log line
```

Comment:
do you mean, when it is structured?

Reply:
I mean when it is a log line generated by the processor - and yes it is structured

Reply:
so write it, like you did above? :)